src/subdomains/staking/domain/entities/withdrawal.entity.ts

```diff
@@ -18,7 +18,7 @@ export class Withdrawal extends IEntity {
   @Column({ nullable: true })
   signature: string;
 
-  @ManyToOne(() => Staking, (staking) => staking.withdrawals, { nullable: true })
+  @ManyToOne(() => Staking, (staking) => staking.withdrawals, { nullable: false })
```

Comment:
could you please also do nullable: false for rewards?

Reply:
True, makes sense